released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.11/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/math-comp/analysis"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+license: "CECILL-C"
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.11" & < "8.15~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-fingroup" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-algebra" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-solvable" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-field" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-finmap" { (>= "1.5.1" & < "1.6~") | (= "dev") }
+  "coq-mathcomp-bigenough" { (>= "1.0.0") | (= "dev") }
```

Comment:
```suggestion
  "coq-mathcomp-bigenough" { (>= "1.0.0") }
```
Due to how opam version comparisons work, the current formulation is redundant. `>="1.0.02"` will always include `dev` and the like.